projects/hslayers/src/components/map-swipe/map-swipe.component.html

```diff
@@ -1,20 +1,30 @@
+<!-- Layer item template -->
+<ng-template #layerItem let-layer="layer">
+  <div class="hs-lm-item-title d-flex align-items-center" style="word-break:break-all; cursor:move">
+    <button type="button" class="btn btn-sm hs-lm-item-visibility m-0 p-0"
+      (click)="hsMapSwipeService.changeLayerVisibility(layer);$event.stopPropagation()"
+      [ngClass]="hsMapSwipeService.getLayerVisibility(layer) ? 'hs-checkmark' : 'hs-uncheckmark'"></button>
```

Comment:
Not good to call getLayerVisibility function on each digest loop. Better to store visibility in some variable and update through observers.